app/views/state_file/questions/md_county/edit.html.erb

```diff
@@ -9,7 +9,21 @@
       <%= f.vita_min_state_file_select(:residence_county, t(".county_html"), @counties, label_class: 'form-label--bold', prompt: t('general.select_prompt')) %>
     </div>
     <div class="white-group">
-      <%= f.vita_min_state_file_select(:subdivision_code, t('.political_subdivision_html'), [], include_blank: true, prompt: t('general.select_prompt')) %>
+      <strong> <%= t('.political_subdivision')%> </strong>
+      <p class="text--help spacing-below-10"> <%= t('.political_subdivision_label_html')%>  </p>
+      <%= f.vita_min_state_file_select(:subdivision_code, "", include_blank: true, prompt: t('general.select_prompt')) %>
```

Comment:
the label for the selector is moved out of the form tag so that there is space for the description